ctapipe/containers.py

```diff
@@ -399,6 +399,21 @@ class ImageParametersContainer(Container):
     )
 
 
+class ImPACTLikelihoodContainer(Container):
+    """
+    Concentrations are ratios between light amount
+    in certain areas of the image and the full image.
+    """
+
+    default_prefix = "ImPACTLikelihood"
+    log_likelihood = Field(
+        nan, "Per telescope likelihood of the ImPACT best fit location"
+    )
+    impact_distance = Field(
```

Comment:
I have a question here: Because both the `ReconstructedEnergyContainer` and `ReconstructedGeometryContainer` have the fields `is_valid`, `goodness_of_fit` and `telescopes`, we get them twice as `_1` and `_2` respectively. There is no difference in these fields between energy and geometry reconstruction, so should one just be left empty or should they both just give the same values?

Reply:
I'd sat that if you reconstruct it, you should set the appropriate values for its  container.

Reply:
yes, it's ok that they are the same for this reconstructor.

Reply:
All fields that appear twice are now also filled twice